cfgov/ask_cfpb/jinja2/ask-cfpb/landing-page.html

```diff
@@ -49,50 +49,789 @@ <h2 class="u-show-on-mobile u-mt45">
+                                <a class="" href="/consumer-tools/student-loans/">
+                                    {{ svg_icon( "college" ) }}
+                                    <span>Student loans</span>
                                 </a>
                             </h2>
 
                             <ul class="m-list">
-                                {% for page in portal.featured_answers %}
+                                
+                                    <li class="m-list_item">
+                                    <a class="a_link a-link__partially-styled" data-pretty-href="cfpb.gov/askcfpb/545" href="/ask-cfpb/what-are-the-different-ways-to-pay-for-college-or-graduate-school-en-545/">
+<span class="a-link__partially-styled_plain">What are the different ways to pay for college or graduate school? </span>
+<span class="a-link__partially-styled_underlined">Read answer</span>
+</a>
+                                    </li>
+                                
                                     <li class="m-list_item">
-                                    {{ partially_styled_link.render({
-                                        'plain_text': page.question,
-                                        'underlined_text': read_answer_text,
-                                        'url': page.url })
-                                    }}
+                                    <a class="a_link a-link__partially-styled" data-pretty-href="cfpb.gov/askcfpb/641" href="/ask-cfpb/what-is-public-service-loan-forgiveness-en-641/">
+<span class="a-link__partially-styled_plain">What is Public Service Loan Forgiveness? </span>
+<span class="a-link__partially-styled_underlined">Read answer</span>
+</a>
+                                    </li>
+                                
+                            </ul>
+
+                            <div class="m-card_footer">
+                                <a href="/consumer-tools/student-loans/">
+                                Learn more about student loans
+                                </a>
+                            </div>
+
+                        </article>
+                        {% else %}
+                        <article class="m-card">
+                            <h2 class="m-card_heading">
+                                <a class="" href="/es/herramientas-del-consumidor/prestamos-para-vehiculos/">
+                                    {{ svg_icon( "car" ) }}
+                                    Préstamos para vehículos
+                                </a>
+                            </h2>
+
+                            <ul class="m-list">
+                                
+                                    <li class="m-list_item">
+                                    <a href="/es/obtener-respuestas/que-efecto-tendria-sobre-mi-credito-comparar-diferentes-ofertas-de-prestamos-para-vehiculos-es-763/" class="a_link a-link__partially-styled"> 
```

Comment:
Do the Spanish pages not have a `data-pretty-href` URL, like the english pages do? This should only come into play when printing.

Reply:
That's actually a good question -- I don't think they do, just based on the fact that I pulled this HTML from the current rendered pages in both cases. Also not seeing anything that would work for Spanish in `redirects.conf` though I might just be missing it...

Reply:
just scanned through a few live pages to compare Spanish and English, and yeah, there's no equivalent shortening for `data-pretty-href` in Spanish.

Reply:
Yeah there's no equivalent pattern to Spanish.